resources/views/settings/edit.blade.php

```diff
@@ -50,7 +50,7 @@
                         @endif
                     </div>
 
-                    <div class="form-group">
+                    <div class="form-group mb-4">
```

Comment:
No... It was only on that point, to separate form group sections Teams and Applications.